import-export-cli/utils/constants.go

```diff
@@ -62,6 +62,7 @@ const defaultAdminApplicationListEndpointSuffix = "api/am/admin/v1/applications"
 const defaultDevPortalApplicationListEndpointSuffix = "api/am/store/v1/applications"
 const defaultDevPortalThrottlingPoliciesEndpointSuffix = "api/am/store/v1/throttling-policies"
 const defaultClientRegistrationEndpointSuffix = "client-registration/v0.16/register"
+const DefaultTokenEndPoint = "https://localhost:9443/oauth2/token"
```

Comment:
I don't think we can hardcode localhost here. It needs to be populated from --publisher or --devportal URLs. I think Its better to have two methods: getTokenEndpointOfPublisher / getTokenEndpointOfDevPortal. And If we are going to access Publisher REST API, better to get a token from Publisher node and vise versa. However, this approach might need a bit of code change. So, shall we use Publisher node' URL to populate the token endpoint as we are accessing publisher in most cases of apictl?